lib/cylc/task_proxy.py

```diff
@@ -707,7 +686,7 @@ def reset_state_unheld(self, stop_point=None):
         self._db_events_insert(event="reset to %s" % (old_status))
         self.log(INFO, 'held => %s' % (old_status))
 
-    def job_submission_callback(self, result):
+    def job_submission_callback(self, result):  # TODO: remove
```

Comment:
Comment removed. It was to be removed, but I have retained it for the moment. Otherwise, `cylc submit` will be very hard to implement.